db/migrate/20210428190634_add_tsvector_index_on_searchable_columnns_to_podcast_episodes.rb

```diff
@@ -0,0 +1,74 @@
+class AddTsvectorIndexOnSearchableColumnnsToPodcastEpisodes < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def up
+    query = <<-SQL
+      ((((
+        to_tsvector('simple'::regconfig, COALESCE(body, ''::text)) ||
+        to_tsvector('simple'::regconfig, COALESCE((subtitle)::text, ''::text))) ||
+        to_tsvector('simple'::regconfig, COALESCE((title)::text, ''::text
+      )))))
+    SQL
```

Comment:
My attempt to make this slightly more digestible 😅

Reply:
Oh yeah, definitely, the thing I posted was pretty gnarly (I copy/pasted straight from the query plan :joy:).

Reply:
Yeah, I don't think there's much else we can do other than space it a little. I copied it as well to be extra safe 🙈 . AFAIK, there isn't any nifty migration syntax to make it "cleaner".

Reply:
Agreed, we could probably remove the explicit type casts and some of the enclosing parens, but it wouldn't be _that_ much better it would still be the same gnarly mess I pasted when you look at the query plan or the index list. 😄 This would be the only place that looked any better and nobody will ever look at it again. :joy:

And it's definitely better in this format, clean enough to understand what's going on — it's concatenating 3 `tsvector`s on `body`, `subtitle`, and `title`.